js/setup.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+var NICNAMES = ['Иван', 'Хуан Себастьян', 'Мария', 'Кристоф', 'Виктор', 'Юлия', 'Люпита', 'Вашингтон'];
+var SURNAMES = ['да Марья', 'Верон', 'Мирабелла', 'Вальц', 'Онопко', 'Топольницкая', 'Нионго', 'Ирвинг'];
+var COLORMANTLES = ['rgb(101, 137, 164)', 'rgb(241, 43, 107)', 'rgb(146, 100, 161)', 'rgb(56, 159, 117)', 'rgb(215, 210, 55)', 'rgb(0, 0, 0)'];
+var COLOR_EYES = ['black', 'red', 'blue', 'yellow', 'green'];
+var NUMBER_WINDOW = 4;
+var usersDialog = document.querySelector('.setup');
+var charactersDialog = document.querySelector('.setup-similar');
+var similarListElement = document.querySelector('.setup-similar-list');
+var similarWizardTemplate = document.querySelector('#similar-wizard-template')
+  .content
+  .querySelector('.setup-similar-item');
+
+/* функция для рандомного создания фамилии и имени */
+var getUserName = function (name, givenName) {
+  var randomName = name[Math.floor(Math.random() * name.length)];
+  var randomFio = givenName[Math.floor(Math.random() * givenName.length)];
+  var common = randomName + ' ' + randomFio;
```

Comment:
Тут можно сократить, но это не обязательно, и я больше проверять это не буду, просто чтобы ты знал, что так можно, но сейчас тебе удобно с переменной - это нормально

```suggestion
  return randomName + ' ' + randomFio;
```